packages/ngx-table/src/lib/services/table-helper.service.ts

```diff
@@ -1,7 +1,7 @@
 import {Injectable} from '@angular/core';
 import {TableColumn} from '../types/table.types';
 
-@Injectable()
+@Injectable({ providedIn: 'root' })
```

Comment:
Please don't use `provideIn`. This will be deprecated in future versions. Dependency inject using declaration in the `NgModule`